app/src/main/res/values/dimens.xml

```diff
@@ -66,4 +66,6 @@
     <dimen name="elevation_submenu_4">12dp</dimen>
     <dimen name="elevation_switch">1dp</dimen>
     <dimen name="elevation_toolbar">4dp</dimen>
+
+    <dimen name="design_navigation_icon_size" tools:override="true">32dp</dimen>
```

Comment:
Holy shit is this how you change the size? I couldn't figure it out for the life of me.